symfony/plugins/orangehrmTimePlugin/Dao/TimesheetDao.php

```diff
@@ -421,16 +425,7 @@ public function getTimesheetItemsForEmployeeReport(
     }
 
     /**
-     * @param  EmployeeReportsSearchFilterParams  $filterParams
-     * @return int
-     */
-    public function getTimesheetItemsCountForEmployeeReport(EmployeeReportsSearchFilterParams $filterParams): int
```

Comment:
Don't format the full file, if you feel your IDE configs are bit different. It's hard to identify actually code change for the particular ticket.

Reply:
Sure